lib/logger.js

```diff
@@ -14,15 +14,16 @@ var ut = require('util');
  */
 
 function viewed(str) {
-  var re = /\u001b\[\d+m/gm;
+  const re = /\u001b\[\d+m/gm;
+
   return String(str).replace(re, '');
 }
 
 function trimTo(str, amt) {
-  var raw = '';
-  var visual = viewed(str).slice(0, amt);
-  var result = '';
-  for (var i = 0; i < str.length; ++i) {
+  const visual = viewed(str).slice(0, amt);
+  let raw = '';
+  let result = '';
+  for (let i = 0; i < str.length; i += 1) {
```

Comment:
@milesj I didn't get any trouble from the linter for this, do you think we could add a linting rule?